buildlib/tools/check_tls_perf_caps.sh

```diff
@@ -8,28 +8,36 @@
 realdir="$(realpath "$(dirname "$0")")"
 source "${realdir}"/../az-helpers.sh
 
-# check that correct UCX versions are used
-LD_LIBRARY_PATH=${UCX_LEGACY_PATH}/lib "${UCX_LEGACY_PATH}"/bin/ucx_info -v
-LD_LIBRARY_PATH=${UCX_PR_PATH}/lib "${UCX_PR_PATH}"/bin/ucx_info -v
-
-LD_LIBRARY_PATH=${UCX_LEGACY_PATH}/lib old_out=$("${UCX_LEGACY_PATH}"/bin/ucx_info -d)
-LD_LIBRARY_PATH=${UCX_PR_PATH}/lib new_out=$("${UCX_PR_PATH}"/bin/ucx_info -d)
+legacy_info_file=ucx_info_legacy.txt
+pr_info_file=ucx_info_pr.txt
 
 export UCX_TCP_BRIDGE_ENABLE=y #bridge devices are not shown by default since v1.16
 
+# Print legacy library versions and configurations
+LD_LIBRARY_PATH=${UCX_LEGACY_PATH}/lib "${UCX_LEGACY_PATH}/bin/ucx_info" -v
+LD_LIBRARY_PATH=${UCX_LEGACY_PATH}/lib "${UCX_LEGACY_PATH}/bin/ucx_info" -d > ${legacy_info_file}
+echo ${legacy_info_file}:
+cat ${legacy_info_file}
+
+# Print new library versions and configurations
+LD_LIBRARY_PATH=${UCX_PR_PATH}/lib "${UCX_PR_PATH}/bin/ucx_info" -v
+LD_LIBRARY_PATH=${UCX_PR_PATH}/lib "${UCX_PR_PATH}/bin/ucx_info" -d > ${pr_info_file}
+echo ${pr_info_file}:
+cat ${pr_info_file}
+
 res=true
-for tl_name in $(echo "${old_out}" | grep Transport | awk '{print $3}')
+for tl_name in $(grep Transport ${legacy_info_file} | awk '{print $3}')
 do
-    old_tl_caps=$(echo "$old_out" | grep -A 8 "Transport: $tl_name")
-    new_tl_caps=$(echo "$new_out" | grep -A 8 "Transport: $tl_name")
-    for device in $(echo "${old_tl_caps}" | grep Device | awk '{print $3}')
+    old_tl_caps=$(grep -A 8 "Transport: $tl_name" ${legacy_info_file} || true)
+    new_tl_caps=$(grep -A 8 "Transport: $tl_name" ${pr_info_file}     || true)
+    for device in  $(echo "${old_tl_caps}" | grep Device | awk '{print $3}')
```

Comment:
```suggestion
    for device in $(echo "${old_tl_caps}" | grep Device | awk '{print $3}')
```

Reply:
per offline discussion will merge as is to unblock CI